src/Components/Aspire.MongoDB.Driver/MongoDbHealthCheckSettings.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.MongoDB.Driver;
+
+/// <summary>
+/// Provides the health check configuration settings for MongoDB databases.
+/// </summary>
+public class MongoDbHealthCheckSettings
```

Comment:
We didn't split this out to a separate type in other components, for example:

https://github.com/dotnet/aspire/blob/c2b1b8fac46468fb3635f7e183a19a27395445dd/src/Components/Aspire.Azure.Messaging.ServiceBus/AzureMessagingServiceBusSettings.cs#L35-L43

Should we just put these 2 properties up in `MongoDbSettings`?

Reply:
Done, please review.